### PR TITLE
examples/dll readme updates

### PR DESCRIPTION
**Pull request by aretecode**
supersedes https://github.com/webpack/webpack/pull/4745 (for cleaner commit history and easier merging)

### CODE REVIEW

File: examples/dll-user/README.md [hunk 1]
```diff
@@ -138,7 +144,7 @@ module.exports = beta_282e8826843b2bb4eeb1;
   \**********************************************************************/
 /***/ (function(module, exports, __webpack_require__) {
 
-module.exports = (__webpack_require__(0))(0);
+module.exports = (__webpack_require__(0))(0)
```
Comment (sokra) [inline examples/dll-user/README.md]: omg the semicolon vanishs. When did we change this...?

Comment (aretecode) [inline examples/dll-user/README.md]: @sokra somewhere between 2.3 & 2.4.1

Comment (sokra) [inline examples/dll-user/README.md]: https://github.com/webpack/webpack/pull/4577#pullrequestreview-33682841

Comment (aretecode) [inline examples/dll-user/README.md]: @sokra ok so when that is merged, run build again?

Comment (sokra) [inline examples/dll-user/README.md]: When this PR is merged: https://github.com/webpack/webpack/pull/4766

Comment (sokra) [inline examples/dll-user/README.md]: merged. Could you rebase this PR?

Comment (aretecode) [inline examples/dll-user/README.md]: @sokra updated

**Comment by sokra** [comment]
Thanks

**Comment by aretecode** [comment]
@sokra you bet, let me know more feedback on the others and if you want to chime in on the js.webpack.org dll docs pr

**Pull request merged**